__tests__/node_fs_test.ml

```diff
@@ -0,0 +1,10 @@
+open Jest
+
+let _ =
+  describe "content" (fun _ ->
+      test "non-zero length" (fun _ ->
+          Just (Truthy (Js.Undefined.bind [%node __filename] begin fun [@bs] f ->
+              let content : string =
+                Node.Fs.readFileSync  f `utf8 in
+              content |> String.length
+            end))))
```

Comment:
I think they are all spaces. I asked ocp-indent to make indentaion for me, and this is result...